apps/studio/components/interfaces/Organization/AuditLogs/AuditLogs.tsx

```diff
@@ -147,6 +139,77 @@ const AuditLogs = () => {
             </div>
           )}
 
+          <div className="flex items-center justify-between">
```

Comment:
The loading state is above the bar
<img width="1271" height="405" alt="Image" src="https://github.com/user-attachments/assets/1daafbc7-b123-44bc-889a-253e301569a4" />